src/codeSigning/packAndSign.ts

```diff
@@ -454,4 +461,11 @@ export const api = {
       }
     }
   },
+
+  getAgentForUri(url: string): false | Agents {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+    const proxyUrl: string = getProxyForUrl(url) as string;
```

Comment:
The function from proxy-from-env takes a target url and then produces a proxy url that should be used. It considers well known proxy env vars and no_proxy to make a decision.